tests/test_models.py

```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
+])
```

Comment:
Include the following test cases/edge cases for a better prepared test function!
Standard deviation for one patient with multiple observations.
Standard deviation for two patients.
Graph includes a standard deviation graph.
Standard deviation function should raise an error if given empty data.
Computing standard deviation where deviation is different from variance.
Standard deviation function should give correct result given negative inputs.
Function should work with numpy arrays.